docs/mysql.md

```diff
@@ -0,0 +1,70 @@
+# Using MySQL
+
+The Docker configuration of this repository is extensible thanks to Flex recipes. By default, the recipe installs PostgreSQL.
+If you prefer to work with MySQL, follow these steps:
+
+First, install the doctrine/orm pack as described: `docker compose exec php composer req symfony/orm-pack`
```

Comment:
```suggestion
First, install the `symfony/orm-pack` package as described: `composer require symfony/orm-pack`
```

Reply:
Installing the `symfony/orm-pack` package should not be done inside a container

Reply:
Normally, it should be inside the container, but in our case it can be a bit tricky.
These files are listed in the `.dockerignore` file but shared in the `compose.override.yaml` and at this stage we are more in dev mode where these files are shared.

Reply:
Maybe I'm misunderstanding you, but isn't it exactly right? Working with composer outside the container is error-prone because the local PHP version does not necessarily match or other parameters are different (extensions, etc.)

Reply:
The Symfony recipe applies to `compose.yml`, `compose.override.yaml`, `Dockerfile`, etc.

If the container is running in development mode, the changes will be applied in the local folder thanks to this line:
https://github.com/dunglas/symfony-docker/blob/f063b11fb8d3522e3811124f43b2145dad205199/compose.override.yaml#L8

Otherwise (production mode) it won't work, so I would recommend using this command outside the container to avoid any confusion. Anyway, this is just a recommendation. Perhaps we should use some uniform style for installing additional packages.

Reply:
The Symfony recipe applies to `compose.yml`, `compose.override.yaml`, `Dockerfile`, etc.

If the container is running in development mode, the changes will be applied in the local folder:
https://github.com/dunglas/symfony-docker/blob/f063b11fb8d3522e3811124f43b2145dad205199/compose.override.yaml#L8

Otherwise (production mode) it won't work, so I would recommend using this command outside the container to avoid any confusion. Anyway, this is just a recommendation. Perhaps we should use some uniform style for installing additional packages.

Reply:
In production mode, it won't work and I'm not sure we'll end up in that this situation, so i'm in favour of doing it inside.
However, it deserves to be mentioned see : https://github.com/dunglas/symfony-docker/pull/571/files

Reply:
You're right @dsentker. The point raised by @bkosun is the files modified by recipes are not sync in the container in prod mode. So the the recipe can't be applied.

Reply:
> However, you raise a good point that deserves to be mentioned see : https://github.com/dunglas/symfony-docker/pull/571/files

Maybe add a second sentence like 

> [!NOTE] Be sure that your host machine does provide  the same requirements as defined in the composer configuration files.